src/stories/Form/component.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+import StyledForm from "./index";
+
+export default {
+    title: "Form",
+    component: StyledForm,
+};
+
+export const Default = () => <StyledForm></StyledForm>;
```

Comment:
No need to export storybook component, this will be automatically available in your storybook.

Reply:
done